redact/index.ts

```diff
@@ -0,0 +1,191 @@
+import { initializeWasm, type SensitiveInfoEntity } from "@arcjet/redact-wasm";
```

Comment:
```suggestion
import { initializeWasm } from "@arcjet/redact-wasm";
import type { SensitiveInfoEntity } from "@arcjet/redact-wasm";
```
While you *can* import types like that and work with verbatimModuleSyntax, I want us to keep them as separate imports.